### ISSUE TITLE
[Bug]: FocusVisible polyfill is not added for sp-picker-button

### ISSUE THREAD

**Issue by deepthi-majji** [adobe/spectrum-web-components#3479]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Picker-button

### Expected behavior

Focus ring styles should be applied on picker button even in browsers that don't support :focus-visible pseudo-class. the 

### Actual behavior

No visible focus ring styles are applied on the picker button  in browsers that don't support :focus-visible pseudo-class. the 

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by blunteshwar** [comment]
It(FocusVisible polyfill is added for sp-picker-button]) is already implemented in the code.
I have informed the person who reported this bug and waiting for his reply!

**Comment by Rajdeepc** [comment]
@deepthi-majji We are awaiting your confirmation on this. Please let us know if this issue still persists or not.

**Comment by deepthi-majji** [comment]
@blunteshwar I can see here in code as well that we are not extending picker-button from FocusVisiblePolyFill: https://github.com/adobe/spectrum-web-components/blob/main/packages/picker-button/src/PickerButton.ts#L38C35-L38C45

**Comment by blunteshwar** [comment]
i believe picker-button extends `buttonBase` which further extends `focusable` which again extends `FocusVisiblePolyFill`

**Comment by Rajdeepc** [comment]
I can confirm that PickerButton is extending the `FocusVisiblePolyFill` mixin. Please let us know if this issue persists.

**Comment by deepthi-majji** [comment]
Okay, we can close this issue.
